DatadogLogs/Sources/Feature/DatadogLogsFeature.swift

```diff
@@ -0,0 +1,64 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+import DatadogInternal
+
+internal let DatadogLogsFeatureName = "logging"
```

Comment:
Yes, I'm also thinking of making the Feature name a static var in the `DatadogFeature` protocol, so then we would do `DatadogLogsFeature.name`, it would also simplify registration. In the meantime, I will commit your second suggestion 👍